### ISSUE TITLE
A "Hilarious" upvote option.

### ISSUE THREAD

**Issue by ssSnapy** [Codewars/codewars.com#3256]
Just what the title says. Would be nice to have a third upvote option for amusing solutions which are neither clever nor best practice.

**Comment by yur-itch** [comment]
Especially on those golfed down solutions, or something like that. It would be very appropriate

**Comment by ZaLiTHkA** [comment]
agreed! purely because [solutions like this](https://www.codewars.com/kata/reviews/615a651ab04c890001d78688/groups/628241800f8bf70001fa9d92) exist... 😆
